api/server/jobs/removeExpiredSessions.js

```diff
@@ -0,0 +1,14 @@
+const cron = require('node-cron');
```

Comment:
I don't think `node-cron` is necessary if we utilize mongoDB TTL indices. You can see how I use it in my latest PR, it's very simple.